### ISSUE TITLE
Bug: Clicking the fullscreen icon bottom left of the screen in the JS example is not working. 

### ISSUE THREAD

**Issue by beautifwhale** [FormidableLabs/spectacle#834]
# Steps to reproduce

Create boilerplate project
```
spectacle-boilerplate \
  --mode "js" \
  --name "deck-o-awesomeness" \
  --description "My fantastic Spectacle presentation" \
--dir presentation
```

Install dependencies
```
cd presentation && npm i
```

Start presentation
```
npm start
```

# Expected

Presentation starts, clicking fullscreen sets the presentation to fullscreen. 

# Actual

Clicking the fullscreen icon bottom left of the screen in the JS boilerplate is not working.

**Comment by ryan-roemer** [comment]
Thanks @beautifwhale !

This actually looks like an upstream issue with spectacle, so we've transferred it here. Issue is reproducible on the auto-presentation deploy here: https://spectacle.formidable.com/?slide=0&slideElement=-1

**Comment by ryan-roemer** [comment]
Oh, and it's a bit confusing, but if you hit other issues with the CLI tool `spectacle-boilerplate`, it's actually part of the `spectacle-cli` package (that provides `spectacle` and `spectacle-boilerplate`). No worries here -- we've found the right home for the issue.

**Comment by kale-stew** [comment]
Closed by #835
